instrumentation/jaxrs-client/jaxrs-client-2.0/jaxrs-client-2.0-common/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/jaxrsclient/v2_0/ClientTracingFilter.java

```diff
@@ -31,6 +31,7 @@ public void filter(ClientRequestContext requestContext) {
   @Override
   public void filter(ClientRequestContext requestContext, ClientResponseContext responseContext) {
     Object contextObj = requestContext.getProperty(CONTEXT_PROPERTY_NAME);
+    requestContext.removeProperty(CONTEXT_PROPERTY_NAME);
```

Comment:
can you add a comment why we are removing the property? (since it's not standard practice across instrumentation to remove properties/attributes that we add - or should it be?)

Reply:
Comment added. If we can easily remove stuff added by us then it would be nice, for example in https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/jaxws/jaxws-2.0-cxf-3.0/library/src/main/java/io/opentelemetry/javaagent/instrumentation/cxf/CxfHelper.java there is a `remove` method that returns the current value so using it instead of `get` doesn't complicate things.